packages/react/src/components/HierarchicalTree/HierarchicalTree.tsx

```diff
@@ -3,7 +3,7 @@ import * as _ from 'lodash'
 import * as PropTypes from 'prop-types'
```

Comment:
Where / how can we mark as depreciated?

Reply:
Created request #1756 to properly handle depreciated methods in DocSite.  For now, can we update text descriptions to make this clear.

Reply:
should we do this in the current PR?

Reply:
In the current PR, just put (DEPRECATED) to the beginning of the description of the Tree. See https://stardust-ui.github.io/react/components/item-layout as an example.

Reply:
We should also provide console warnings when using deprecated components/props once we ship the new Tree.